cves/CVE-2017-7659.yml

```diff
@@ -115,11 +121,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
-  contest:
+  answer: |
+    Someone named Robert Swiecki reported the issue to the security team. The
```

Comment:
Did the security team discuss how it was found? Also, did Robert Swiecki write a blog post about his vulnerability-hunting process somewhere? I found a blog post written by the credited reporter for one of my vulnerabilities that helped shed light on the reporter's fuzzing process that he used to find the CVE.

Reply:
I would have added that had I found it during my research.